trappy/nbexport/templates/trappy.tpl

```diff
@@ -0,0 +1,9 @@
+{% extends 'full.tpl'%}
+{% block html_head %}
+<script src="http://localhost:8080/EventPlot.js" type="text/javascript" charset="utf-8"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/d3/3.5.6/d3.min.js", type="text/javascript" charset="utf-8"></script>
+<script src="http://labratrevenge.com/d3-tip/javascripts/d3.tip.v0.6.3.js" type="text/javascript" charset="utf-8"></script>
+<script src="https://cdn.rawgit.com/sinkap/a98037133fb79195583a1aa846671c22/raw/c23c231cbb1cdfe29eb84b1f786090a13c16dd93/EventPlot.js" type="text/javascript" charset="utf-8">
```

Comment:
If you can inline it, it would be great! But this would require you to
change the notebook template. Which, I honestly think is more difficult to
control. It would need the users to use a custom template.

The CDN link can be created when a release is made via the gist API, which
I think should be manageable.

On Tue, Apr 26, 2016 at 1:43 PM, Javi Merino <notifications@github.com>
wrote:

> In trappy/nbexport/templates/trappy.tpl
> <https://github.com/ARM-software/trappy/pull/182#discussion_r61071087>:
>
> > @@ -0,0 +1,9 @@
> > +{% extends 'full.tpl'%}
> > +{% block html_head %}
> > +<script src="http://localhost:8080/EventPlot.js" type="text/javascript" charset="utf-8"></script>
> > +<script src="https://cdnjs.cloudflare.com/ajax/libs/d3/3.5.6/d3.min.js", type="text/javascript" charset="utf-8"></script>
> > +<script src="http://labratrevenge.com/d3-tip/javascripts/d3.tip.v0.6.3.js" type="text/javascript" charset="utf-8"></script>
> > +<script src="https://cdn.rawgit.com/sinkap/a98037133fb79195583a1aa846671c22/raw/c23c231cbb1cdfe29eb84b1f786090a13c16dd93/EventPlot.js" type="text/javascript" charset="utf-8">
>
> This is a bad idea, we now have two copies of EventPlot.js that have to
> be kept in sync. I'm looking into inlining EventPlot.js to the notebook
> instead of using require to load it. With that we won't need this line.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/ARM-software/trappy/pull/182/files/cb4e427d4b956f0c687b5f0cbb3cc3ad16f64578..f37abd9a293fdcd902f27efb6324255d4ae03169#r61071087>
>